### PR TITLE
Make script/test more robust 

### PR DESCRIPTION
**Pull request by flosell**
This makes the test-script more robust, e.g. when dealing with different golang versions or golint not installed ([e.g. in travis-ci](https://travis-ci.org/rodaine/hclencoder/builds/490696357#L456))

(originally suggested by @promiseofcake in #8)

### CODE REVIEW

**Comment by flosell** [comment]
Travis CI fails because it's running Go 1.6 which is [no longer supported by `golint`](https://github.com/golang/lint/issues/400). Should we bump the Go version for Travis CI?

**Comment by promiseofcake** [comment]
😓I think that's reasonable, on the other hand we can check the go-version in script/test if we _really_ wanted to support 1.6 and use the built-in in that case. But I will defer to @rodaine

**Comment by rodaine** [comment]
Oh that's just bit rot. Feel free to update it to 1.11; if we want to get fancy a variable matrix could be used for a handful of versions but the code is so vanilla I don't think it's necessary.

**Comment by flosell** [comment]
Hmm, jumping from one problem to the next :-) `go vet` is more strict in 1.11... I'll have a deeper look at this tomorrow...

**Comment by flosell** [comment]
Ok, all checks green now on 1.11, ready to merge if you want it

**Review by rodaine — APPROVED** [review]

**Comment by rodaine** [comment]
Thanks!

**Pull request merged**
